service/history/tasks/workflow_task.go

```diff
@@ -17,6 +17,7 @@ type (
 		TaskQueue           string
 		ScheduledEventID    int64
 		Version             int64
+		Stamp               int32
```

Comment:
Is my thinking right that only the workflow transfer and timeout task need the stamp?

Reply:
For workflow pause workstream we might need it in `UserTimerTask` & `StartChildExecutionTask` as well. But we are not ready for it yet. So we can add it in the future if needed.